src/main/java/io/confluent/connect/jdbc/TableMonitorThread.java

```diff
@@ -102,7 +104,7 @@ private boolean updateTables() {
     synchronized (db) {
       final List<String> tables;
       try {
-        tables = JdbcUtils.getTables(db);
+        tables = JdbcUtils.getTables(db,table_types);
```

Comment:
If we're going to allow setting this, should we also support an empty list that gets all types? This might require a small change to `JdbcUtils.getTables()`, but otherwise seems simple to support.